### PR TITLE
Implement Reset password UI

### LINKED ISSUE #28: Implement Reset password UI
[Figma Design](https://www.figma.com/design/d60yqkXZspwUMpdiyyFyZn/GlowUp-application?node-id=229-3970&node-type=frame&t=TRiibIHSioJCNBzb-0)

### PR DESCRIPTION
**Pull request by truongtdph40209**
## Related issues
Resolved #28 

## Summary
- Implement ResetPassword screen includes: 
   + Enter new password and confirm
   + Dark mode, light mode
- Update translation key 
- Add router ResetPassword and ForgotPassword

### Steps:
1. Open `Index.js` and delete the code in render
2. **Paste** the following line:

```jsx
<ResetPasswordTemplate/>
```
3. Run app

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|image|image|

### CODE REVIEW

**Comment by NguyenVanViet2004** [comment]
Simulator Screenshot - iPhone 15 Pro - 2024-10-08 at 20 52 54


where is the back icon and icon eye for the hidden password?

File: components/templates/ResetPasswordTemplate.tsx
Comment (KMTus) [inline components/templates/ResetPasswordTemplate.tsx:1]: We need a function to check the right or wrong conditions of the password, interface or notice if they do not match (can be used to display status or use modal)

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]
Update the password reset image in this PR content

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]
API?

**Review by NguyenVanViet2004 — APPROVED** [review]

**Pull request merged**